distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/State.enso

```diff
@@ -12,7 +12,7 @@ from project.Errors.Common import Uninitialized_State
 
    Arguments:
    - key: The key to associate your local_state with in the environment.
-     It is recommended that types be used as keys.
+     Use types as keys.
```

Comment:
The signature of `run : Any -> Any -> Any -> Any` remains unchanged, as we don't have any _type_ for `type Some_Type`.